daemon/daemon.go

```diff
@@ -247,6 +247,11 @@ func (daemon *Daemon) restore() error {
 					logrus.WithError(err).Errorf("Failed to delete container %s from containerd", c.ID)
 					return
 				}
+			} else if daemon.configStore.LiveRestoreEnabled {
```

Comment:
if !daemon.configStore.LiveRestoreEnabled, am I missing something?

Reply:
Yes, you are correct...